packages/dialect-react-ui/components/Chat/screens/Main/pages/CreateThreadPage/CreateThread.tsx

```diff
@@ -74,6 +75,48 @@ function ActionCaption({
   return null;
 }
 
+const showAddressInputStatus = (
+  valid: boolean,
+  address: string,
+  cardinalAddress: string
+) => {
+  const { textStyles } = useTheme();
+
+  if (!address) {
+    return (
+      <P className={clsx(textStyles.small, 'dt-text-red-500 dt-mt-1 dt-px-2')}>
+        Empty address or twitter handle
+      </P>
+    );
+  }
+
+  const isTwitter = address.charAt(0) === '@';
+
+  if (isTwitter && valid && cardinalAddress) {
+    return (
+      <P
+        className={clsx(textStyles.small, 'dt-text-green-500 dt-mt-1 dt-px-2')}
+      >
+        Associated address is: {cardinalAddress}
+      </P>
+    );
+  } else if (isTwitter && !valid && !cardinalAddress) {
```

Comment:
Since we're doing returns, there is no need to `else if`

```
if (...) { return ...; }

if (...) { return ...; }

...
```

Hope it makes sense.

Reply:
forgot to update here that changes have been made :)